test/js/ez-browse.js

```diff
@@ -0,0 +1,95 @@
+describe('ez-browse', function() {
+    let element;
+
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+    });
+
+    it('should be defined', function () {
+        assert.equal(
+            window.customElements.get('ez-browse'),
+            element.constructor
+        );
+    });
+
+    describe('properties', function () {
+        describe('`selectedLocationId`', function () {
+            it('should be set by the attribute', function () {
+                const value = '42';
+
+                element.setAttribute('selected-location-id', value);
+                assert.equal(element.selectedLocationId, value);
+            });
+        });
+    });
+
+    describe('content discover', function () {
```

Comment:
in those tests you are testing what's happening on click, so here you should write:
```js
describe('click', function () {
```